uav_simulator/MapParam param
uav_simulator/SensorParam sparam
---
uav_simulator/State state
bool success